tools/osbuilder/rootfs-builder/ubuntu/config.sh

```diff
@@ -7,6 +7,7 @@ OS_NAME=ubuntu
 OS_VERSION=${OS_VERSION:-focal}
 PACKAGES="chrony iptables"
 [ "$AGENT_INIT" = no ] && PACKAGES+=" init"
+[ "$KATA_BUILD_CC" = yes ] && PACKAGES+=" cryptsetup-bin e2fsprogs"
```

Comment:
While adding the `KATA_BUILD_CC` here is fine, I'd like to see this propagated to the scripts we use to generate the image.

So, you'd need to adapt the following scripts to ensure we have `KATA_BUILD_CC` set.
* https://github.com/kata-containers/kata-containers/blob/81e065f7bdb741ef4bd2106c19067f795bbbdb06/tools/packaging/kata-deploy/local-build/kata-deploy-binaries-in-docker.sh#L47
* https://github.com/kata-containers/kata-containers/blob/81e065f7bdb741ef4bd2106c19067f795bbbdb06/tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh#L118

Reply:
Thanks for the reminder, fixed now!